### PR TITLE
Update documentation

### PR DESCRIPTION
**Pull request by danshick**
Created FAQ, compatibility information, and contribution guidelines. Updated README.

### CODE REVIEW

File: docs/FAQ.md [hunk 1]
```diff
@@ -0,0 +1,103 @@
...
+## What are xdg-desktop-portals?
+
+They were originally designed to offer access to system resources inside of
+flatpak sandboxes. More recently, large compositors have influenced many
+software makers to adopt these portals as standard mechanisms for screenshot
+and screencast functionalities in Wayland.
+
+## What is PipeWire?
+
+[PipeWire] s a server and user space API to deal with multimedia pipelines.
```
Comment (soyuka) [inline docs/FAQ.md:18]:
```suggestion
[PipeWire] is a server and user space API to deal with multimedia pipelines.
```

File: docs/screencast-compatibility.md [hunk 2]
```diff
@@ -0,0 +1,124 @@
...
+No matter what browser you are using, the WebRTC code will check that the
+environment variable  ```XDG_SESSION_TYPE=wayland``` is set
+in order to function.
+
+Additionally, there is a compatibility issue related to pixelformats that
+currently requires the flag ```-p BGRx``` to be added to xdg-desktop-portal-wlr
+in order to get WebRTC to correctly negotiate a pipewire stream. Support for
+additional pixel formats in WebRTC will fix this issue, but we are also
+discussing being more liberal about the formats we claim. You can read more
+about that [here](https://github.com/emersion/xdg-desktop-portal-wlr/issues/23).
```
Comment (soyuka) [inline docs/screencast-compatibility.md:27]: WDYT about adding a link to the webrtc test available on this page: https://mozilla.github.io/webrtc-landing/gum_test.html?

Comment (danshick) [reply docs/screencast-compatibility.md:27]: That's a good idea. I'll link it before we commit.

**Comment by danshick** [comment]
At colemickens's suggestion on IRC, I also want to update this to be a little clearer about the role of the wlr.portal file and how xdp starts xdpw.

File: docs/FAQ.md [hunk 3]
```diff
@@ -0,0 +1,103 @@
...
+reason xdpw cannot create multiple simultaneous streams of different outputs.
+
+## Will this let me share my screen in Zoom?
+
+For the web-based client, yes it will. For the native linux app, no, they use a
+Gnome specific API that this project does not seek to emulate.
+
+If this is something you care about, please leave [feedback for zoom], suggest
+an improvement for the Linux client, and ask them to support the
+xdg-desktop-portal screencast API instead of the proprietary Gnome one.
```
Comment (hedgepigdaniel) [inline docs/FAQ.md:54]: proprietary sounds like its not open source... gnome specific / non standard is perhaps more fair?

Comment (emersion) [reply docs/FAQ.md:54]: I'd prefer "private". I also think "proprietary" is fine in this context.

Comment (foxcpp) [reply docs/FAQ.md:54]: https://www.thefreedictionary.com/proprietary
Yes. Non-standard is the correct word here.

**Comment by danshick** [comment]
Also want to mention possibly needing to restart pipewire sometimes. Still can't get a good log or backtrace when that happens.

File: docs/FAQ.md (hunk 3 above)
Comment (danshick) [reply docs/FAQ.md:54]: Yeah, I can see this both ways. I think I like private too.

Comment (emersion) [reply docs/FAQ.md:54]: It's private, as in not intended for third-party use, reserved for internal GNOME mechanisms.

File: docs/FAQ.md [hunk 4]
```diff
@@ -0,0 +1,103 @@
...
+
+They were originally designed to offer access to system resources inside of
+flatpak sandboxes. More recently, large compositors have influenced many
+software makers to adopt these portals as standard mechanisms for screenshot
+and screencast functionalities in Wayland.
+
+## What is PipeWire?
+
+[PipeWire] s a server and user space API to deal with multimedia pipelines.
+It's scope is broad, seeking to provide a single mechanism for transporting
```
Comment (hugopeixoto) [inline docs/FAQ.md:19]:
```suggestion
Its scope is broad, seeking to provide a single mechanism for transporting
```

**Comment by danshick** [comment]
@emersion and I decided to move some of this to the project wiki. I'll still be incorporating suggestions from here, but this PR will change as some of the files are getting moved to there.

File: docs/FAQ.md (hunk 4 above)
Comment (danshick) [reply docs/FAQ.md:19]: Ugh, good catch.

**Pull request merged**